helm: remove csi RBAC from the operator

Madhu-1 opened the pull request:
CSI template and RBAC's are shipped as another charts. we dont need to add the csi RBAC to the operator charts.

Madhu-1 commented:
> What chart is the rbac in? Does Rook need to add a helm dependency on it or does the csi operator chart have a dependency on it?

@travisn the CSI RBAC was part of both charts. am removing it from the operator chart so that it will be present only in the csi-driver charts

Madhu-1 committed:
helm: remove csi RBAC from the operator

CSI template and RBAC's are shipped
as another charts. we dont need to add the
csi RBAC to the operator charts.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

Madhu-1 commented:
@travisn @subhamkrai are we good to merge this one?

subhamkrai commented:
> @travisn @subhamkrai are we good to merge this one?

Yeah, looks good to me with all the Rook testing

Madhu-1 committed:
ci: update helmify version

updating the helmify version to the
latest .20 release.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

Madhu-1 committed:
ci: add generated yamls

after the helmify is updated to new version
there are new fields added, add those generated
updated to the repo.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>
Co-authored-by: Allen Conlon <software@conlon.dev>

Madhu-1 committed:
helm: add support for priorityclass

Add support to set the priorityClass
to the operator deployment.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>
Co-authored-by: Rewant Soni <resoni@redhat.com>
